geopyspark/geotrellis/rdd.py

```diff
@@ -48,7 +49,41 @@ def _reclassify(srdd, value_map, data_type, boundary_strategy, replace_nodata_wi
             return srdd.reclassifyDouble(new_dict, boundary_strategy, replace_nodata_with)
 
 
-class RasterRDD(object):
+class RDDWrapper(object):
```

Comment:
Could you also add a docstring for this class? Something like this https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark/geotrellis/rdd.py#L52